src/Stratis.Bitcoin/P2P/PeerConnectorConnect.cs

```diff
@@ -70,8 +70,41 @@ public override void OnStartConnect()
 
                 PeerAddress peerAddress = this.peerAddressManager.FindPeer(ipEndpoint);
                 if (peerAddress != null && !this.IsPeerConnected(peerAddress.Endpoint))
-                    await ConnectAsync(peerAddress).ConfigureAwait(false);
+                {
+                    // Nodes disallow connection to peers in same range to prevent sybil attacks.     
+                    if (ipEndpoint.Address.IsLocal())
+                    {
+                        // Local peer: filtering is disabled unless explicitly set to true.
```

Comment:
No need for this behaviour, just always ignore for local addresses

Reply:
Done.